temporalcli/commands.gen.go

```diff
@@ -229,13 +230,15 @@ func NewTemporalServerStartDevCommand(cctx *CommandContext, parent *TemporalServ
 	s.Command.Flags().IntVar(&s.MetricsPort, "metrics-port", 0, "Port for /metrics. Default is off.")
 	s.Command.Flags().IntVar(&s.UiPort, "ui-port", 0, "Port for the Web UI. Default is --port + 1000.")
 	s.Command.Flags().BoolVar(&s.Headless, "headless", false, "Disable the Web UI.")
-	s.Command.Flags().StringVar(&s.Ip, "ip", "127.0.0.1", "IP address to bind the frontend service to.")
+	s.Command.Flags().StringVar(&s.Ip, "ip", "localhost", "IP address to bind the frontend service to.")
 	s.Command.Flags().StringVar(&s.UiIp, "ui-ip", "", "IP address to bind the Web UI to. Default is same as --ip.")
 	s.Command.Flags().StringVar(&s.UiAssetPath, "ui-asset-path", "", "UI custom assets path.")
 	s.Command.Flags().StringVar(&s.UiCodecEndpoint, "ui-codec-endpoint", "", "UI remote codec HTTP endpoint.")
 	s.Command.Flags().StringArrayVar(&s.SqlitePragma, "sqlite-pragma", nil, "Specify SQLite pragma statements in pragma=value format.")
 	s.Command.Flags().StringArrayVar(&s.DynamicConfigValue, "dynamic-config-value", nil, "Dynamic config value, as KEY=JSON_VALUE (string values need quotes).")
 	s.Command.Flags().BoolVar(&s.LogConfig, "log-config", false, "Log the server config being used in stderr.")
+	s.LogLevelServer = NewStringEnum([]string{"debug", "info", "warn", "error", "off"}, "warn")
+	s.Command.Flags().Var(&s.LogLevelServer, "log-level-server", "Log level for the server only.")
```

Comment:
Not related to this PR but I think it would be nice to show the user the available alternative enum values

```
      --log-level-server string            Log level for the server only. (default "warn")
```

Reply:
Does `--log-level off` have precedent elsewhere in Temporal/Go? If so then fine; if not, then I'll also throw in `--log-level never` as a candidate, since it's sort of consistent gramatically: `debug, info, warn, error, never`.

Reply:
> Not related to this PR but I think it would be nice to show the user the available alternative enum values

Agreed. We currently have a general concept of trailing sentences being removed from the English part, and Cobra doesn't support any kind of enum listing. So we can, at code writing time, append the "Accepted values: <values...>" for all enums.

> Does --log-level off have precedent elsewhere in Temporal/Go?

No, only here I think, so I'd support changing this to `never` for for sure.

I can open a PR for both of these shortly...